packages/analytics-react-native/src/index.ts

```diff
@@ -0,0 +1,24 @@
+export {
+  add,
+  flush,
+  getDeviceId,
+  getSessionId,
+  getUserId,
+  groupIdentify,
+  identify,
+  init,
+  logEvent,
+  remove,
+  revenue,
+  setDeviceId,
+  setGroup,
+  setOptOut,
+  setSessionId,
+  setTransport,
+  setUserId,
+  track,
+} from './react-native-client';
+export { runQueuedFunctions } from './utils/snippet-helper';
+export { Revenue, Identify } from '@amplitude/analytics-core';
+import * as Types from '@amplitude/analytics-types';
+export { Types };
```

Comment:
Typescript was giving me a weird error here, and for some reason this worked to avoid the error. I'll go back and check if I can revert this change 👍